<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtraParamDialog</class>
 <widget class="QWidget" name="ExtraParamDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="filter_lineEdit">
     <property name="placeholderText">
      <string>Filter</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="params_listView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="add_pushButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Add the selected parameters.</string>
     </property>
     <property name="text">
      <string>Add parameter</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="addAll_pushButton">
     <property name="toolTip">
      <string>Add all available parameters.</string>
     </property>
     <property name="text">
      <string>Add all known parameters</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="customParameter_pushButton">
     <property name="toolTip">
      <string>Add a custom command line parameter to the minizinc command or to the solver executable.</string>
     </property>
     <property name="text">
      <string>Add custom parameter</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
